functions/views/culturalWeek.ejs

```diff
@@ -205,8 +205,8 @@
 								<h5>Management Week</h5>
 							</a>
 						</div>
-						<div
-							class="blog-single-content__paging__divide blog-single-content__paging__divide--next"
+<!--						<div
+							class="blog-single-content__paging__divide blog-single-content__paging__divide&#45;&#45;next"
```

Comment:
```suggestion
							class="blog-single-content__paging__divide blog-single-content__paging__divide--next"
```

@suneel87 Don't use HTML entities here. If you have some extension for that, disable it as it's clearly not working correctly.

Reply:
Wha...I don't remember putting that there..I didn't even touch that, all I did was comment out a portion of the code